nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -97,9 +98,12 @@ public static void checkInstantiationForParameterizedTypedTree(
             upperBound.getAnnotationMirrors();
         boolean hasNullableAnnotation =
             Nullness.hasNullableAnnotation(annotationMirrors.stream(), config);
+        LibraryModelsHandler libHandler = new LibraryModelsHandler(config);
+        boolean hasNullableUpperBoundLibModel =
+            libHandler.nullableUpperBoundExistsInLibModel(baseType.tsym.toString(), i);
```

Comment:
We don't want to create a new `LibraryModelsHandler` object here.  To be consistent with the rest of the code, I think what we want here is to add a new method to the `Handler` interface like `onOverrideTypeParameterUpperBound`.  We can implement that method in `LibraryModelsHandler`.  Then we could pass in the `handler` object from the `NullAway` class and then call `handler.onOverrideTypeParameterUpperBound` here.

Reply:
I agree, I didn't want to add the method to the interface till I was sure this is the right approach. I'll add it now.